### PR TITLE
Feature test cases

### PR DESCRIPTION
**Pull request by sachind-cuelogic**
Test cases of :
- Sign Up success and fail
- Login success and fail
- Create Project 
- Successfully redirect

### CODE REVIEW

File: bms/bms_app/models.py [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
 
-class User_info(models.Model):
-	username = models.CharField(max_length=50)
-	email    = models.EmailField(unique=True, blank=False)
-	password = models.CharField(max_length=50)
-	designation = models.CharField(max_length=50,default="Software Engineer")
-	is_active   = models.BooleanField(default=True)
-	is_admin    = models.BooleanField(default=False)
-	is_staff    = models.BooleanField(default=False)
+    def __str__(self):
```
Comment (ananthu-nair707) [inline bms/bms_app/models.py]: Why 2 __str__(self) functions for same model class?

File: bms/bms_app/static/bms_app/js/prod_list.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+$(document).ready(function() {
+    $("#menu-toggle").click(function(e) {
+        e.preventDefault();
+        $("#wrapper").toggleClass("toggled");
+    });
+
+});
+
+$(document).ready(function(){
```
Comment (ananthu-nair707) [inline bms/bms_app/static/bms_app/js/prod_list.js]: Why 2 different document ready functions?

File: bms/bms_app/tests.py [hunk 3]
```diff
@@ -1,48 +1,85 @@
...
-	  	temp_pass = os.environ.get('password')
-	  	regexp = re.compile('^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/')	
-	  	if regexp.search(temp_pass):
-	  		self.assertTrue(temp_pass)
-	  	else:
-	  		self.assertFalse(temp_pass)
+class SimpleTest(TestCase):
+
+	def test_login(self):
+		user = User.objects.create_user('mahesh', 'mahesh@mail.com','mahesh123')
```
Comment (ananthu-nair707) [inline bms/bms_app/tests.py]: Remove hardcoded values.

File: bms/bms_app/views.py [hunk 4]
```diff
@@ -9,39 +13,108 @@
...
-def product_home_page(request):
-    return render(request, 'registration/home_page.html')
-    
\ No newline at end of file
+def create_product(request):
+    if request.method == 'GET':
+        users = User.objects.filter()
+        roles = UserRole.objects.filter()
+        prod_types = Product_type.objects.filter()
+        return render(request, 'registration/create_product.html', {'users': users, 'roles': roles, 'prod_types': prod_types})
```
Comment (ananthu-nair707) [inline bms/bms_app/views.py]: 80 char rule.

**Review by ananthu-nair707 — CHANGES_REQUESTED** [review]
Please make these final changes to merge this code.

File: bms/bms_app/models.py (hunk 1 above)
Comment (sachind-cuelogic) [reply bms/bms_app/models.py]: I forgotted to remove First str(self) functions

File: bms/bms_app/static/bms_app/css/create_prodStyle.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
+
+.createprod
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/create_prodStyle.css]: Follow CSS naming conventions, and write reusable CSS, for reference check below link.
https://blog.toughbyte.com/blabla-9fd86eae4e6c

File: bms/bms_app/static/bms_app/css/create_prodStyle.css [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+{
+    padding-left: 100px;
+}
+#navbrand
+{
+    padding-left: 200px;
+}
+#cancelbuton
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/create_prodStyle.css]: Two words should be seperated by  "-" eg. cancel-button.
Button spelling is wrong

File: bms/bms_app/static/bms_app/css/create_prodStyle.css [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+{
+    margin-right: 20px;
+}
+
+#fileForm
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/create_prodStyle.css]: Camel captioning should not be used in CSS.

File: bms/bms_app/static/bms_app/css/land_common.css [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+#footer > .container {
+    padding-top: 5px padding-left: 15px;
+    padding-right: 15px;
+}
+
+.center-container {
+    height: 100%;
+    display: table;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/land_common.css]: Display: table always takes 100% width, you don't have to specify it again.

File: bms/bms_app/static/bms_app/css/prod_list.css [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+    transition: all 0.5s ease;
+}
+
+#wrapper.toggled {
+    padding-left: 250px;
+}
+
+#sidebar-wrapper {
+    position: fixed;
+    left: 250px;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/prod_list.css]: you have given   left: 250px and margin:-250px, which is unnecessary

File: bms/bms_app/static/bms_app/css/prod_list.css [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+    font-size: 18px;
+    line-height: 60px;
+}
+
+.sidebar-nav > .sidebar-brand a {
+    color: #999999;
+}
+
+.sidebar-nav > .sidebar-brand a:hover {
+    background: none;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/prod_list.css]: background:none is not valid CSS property.

File: bms/bms_app/static/bms_app/css/prod_list.css [hunk 11]
```diff
@@ -0,0 +1,219 @@
...
+}
+
+.panel-body {
+    padding: 30px;
+}
+
+#panelinfo {
+    margin-top: 30px;
+    height: 500px;
+    overflow-y: scroll;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/prod_list.css]: for better user experience overflow should be given "auto" instead of "scroll"

**Review by ananthu-nair707 — APPROVED** [review]

**Pull request merged**